hrm/.gitignore

```diff
@@ -0,0 +1,5 @@
+# MacOS
```

Comment:
Why not just add this to the `.gitignore` in the project root and remove this file?

Reply:
Removed and added to `.gitignore` in project root